app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -137,6 +137,24 @@ def rerun
         end
       end
 
+      api :GET, '/job_invocation/:id/outputs', N_('Get outputs of hosts in a job')
```

Comment:
The first time around I though "why have I used singular here, we are not github", second time "something is odd".
I've figured that while checking the `/apidocs` Turns out I've been following the documentation without even knowing it xD
```suggestion
      api :GET, '/job_invocations/:id/outputs', N_('Get outputs of hosts in a job')
```